kratos/utilities/variable_utils.cpp

```diff
@@ -56,24 +56,14 @@ array_1d<double, 3> VariableUtils::SumNonHistoricalNodeVectorVariable(
 {
     KRATOS_TRY
 
+    using ReductionType = typename std::conditional< std::is_scalar<array_1d<double, 3>>::value , SumReduction<double> , Array3Reduction >::type;
+
```

Comment:
Maybe I missed something, but isn't the result of this always `Array3Reduction`?